main.ts

```diff
@@ -969,7 +1101,186 @@ const issueCommand = new Command()
+    "Parent issue (if any) as a team_number code",
+  )
+  .option(
+    "--dry-run",
+    "Dry run: do not create the issue, print the mutation",
+  )
+  .option(
+    "--priority <priority:number>",
+    "Priority of the issue (1-4, descending priority)",
+  )
+  .option(
+    "--estimate <estimate:number>",
+    "Points estimate of the issue",
+  )
+  .option(
+    "-l, --labels [labels...:string]",
+    "Issue labels associated with the issue. May be repeated.",
+  )
+  .option(
+    "--team <team:string>",
+    "Team associated with the issue (if not your default team)",
+  )
+  .option(
+    "--project <project:string>",
+    "Name of the project with the issue",
+  )
+  .option(
+    "--no-use-default-template",
+    "Do not use default template for the issue",
+  )
+  .option("--no-color", "Disable colored output")
+  .arguments("[title:string]")
+  .action(
+    async (
+      {
+        start,
+        assignee,
+        dueDate,
+        useDefaultTemplate,
+        parent,
+        priority,
+        estimate,
+        labels,
+        team,
+        project,
+        dryRun,
+        color,
+      },
+      title,
+    ) => {
+      const showSpinner = color && Deno.stdout.isTerminal();
+      const spinner = showSpinner ? new Spinner() : null;
+      spinner?.start();
+      try {
+        const teamId = (team === undefined)
+          ? await getTeamId()
+          : team.toUpperCase();
+        const teamUid = await getTeamUid(teamId, spinner);
+        if (start && assignee === undefined) {
+          assignee = "self";
+        }
+        if (start && assignee !== undefined && assignee !== "self") {
+          console.error("Cannot use --start and a non-self --assignee");
+        }
+        let assigneeId = undefined;
+        if (assignee !== undefined) {
+          try {
+            // try with display name first
+            assigneeId = await getUserId(assignee, spinner);
+          } catch (_) {
+            assigneeId = await tryPartialMatch(
+              "user",
+              "name",
+              assignee,
+              true,
+              spinner,
+            );
+          }
+        }
+        const labelIds: string[] = [];
+        if (labels !== undefined && labels !== true && labels.length > 0) {
+          // sequential in case of questions
+          for (const label of labels) {
+            labelIds.push(await getIssueLabelId(label, spinner));
+          }
+        }
+        const projectId = (project !== undefined)
+          ? await getProjectId(project, spinner)
+          : undefined;
+        const parentId = (parent !== undefined)
+          ? await getIssueId(parent)
+          : undefined;
+        if (dueDate !== undefined) {
+          const parsed = new Date(dueDate);
+          if (isNaN(parsed.getTime())) {
+            console.error("Invalid due date format");
+            Deno.exit(1);
+          }
+          dueDate = parsed.toISOString().substring(0, 10);
+        }
```

Comment:
if we just avoid date parsing all together here, does the graphql error from a poorly formed date look ok? i.e. it has an error message about the date field.

if so, can we just let the date validation be handled in the graphql request and avoid parsing it here?

Reply:
Will experiment tomorrow.

Reply:
Yes, a bad date is refused by graphql. Removed.